src/test/regress/expected/local_shard_execution.out

```diff
@@ -1700,129 +1700,11 @@ NOTICE:  executing the command locally: SELECT key, ser, ts, collection_id, valu
-      4
-(1 row)
-
-EXECUTE serial_prepared_local;
-NOTICE:  executing the command locally: INSERT INTO local_shard_execution.collections_list_1470009 (key, ser, collection_id) VALUES ('5'::bigint, '3940649673949187'::bigint, 0) RETURNING key, ser
- key |       ser
----------------------------------------------------------------------
-   5 | 3940649673949187
-(1 row)
-
-SELECT setval('collections_list_key_seq', 5);
- setval
----------------------------------------------------------------------
-      5
-(1 row)
-
-EXECUTE serial_prepared_local;
-NOTICE:  executing the command locally: INSERT INTO local_shard_execution.collections_list_1470011 (key, ser, collection_id) VALUES ('6'::bigint, '3940649673949188'::bigint, 0) RETURNING key, ser
- key |       ser
----------------------------------------------------------------------
-   6 | 3940649673949188
-(1 row)
-
-SELECT setval('collections_list_key_seq', 499);
- setval
----------------------------------------------------------------------
-    499
-(1 row)
-
-EXECUTE serial_prepared_local;
-NOTICE:  executing the command locally: INSERT INTO local_shard_execution.collections_list_1470011 (key, ser, collection_id) VALUES ('500'::bigint, '3940649673949189'::bigint, 0) RETURNING key, ser
- key |       ser
----------------------------------------------------------------------
- 500 | 3940649673949189
-(1 row)
-
-SELECT setval('collections_list_key_seq', 700);
- setval
----------------------------------------------------------------------
-    700
-(1 row)
-
-EXECUTE serial_prepared_local;
-NOTICE:  executing the command locally: INSERT INTO local_shard_execution.collections_list_1470009 (key, ser, collection_id) VALUES ('701'::bigint, '3940649673949190'::bigint, 0) RETURNING key, ser
- key |       ser
----------------------------------------------------------------------
- 701 | 3940649673949190
-(1 row)
-
-SELECT setval('collections_list_key_seq', 708);
- setval
----------------------------------------------------------------------
-    708
-(1 row)
-
-EXECUTE serial_prepared_local;
-NOTICE:  executing the command locally: INSERT INTO local_shard_execution.collections_list_1470011 (key, ser, collection_id) VALUES ('709'::bigint, '3940649673949191'::bigint, 0) RETURNING key, ser
- key |       ser
----------------------------------------------------------------------
- 709 | 3940649673949191
-(1 row)
-
-SELECT setval('collections_list_key_seq', 709);
- setval
----------------------------------------------------------------------
-    709
-(1 row)
-
-EXECUTE serial_prepared_local;
-NOTICE:  executing the command locally: INSERT INTO local_shard_execution.collections_list_1470009 (key, ser, collection_id) VALUES ('710'::bigint, '3940649673949192'::bigint, 0) RETURNING key, ser
- key |       ser
----------------------------------------------------------------------
- 710 | 3940649673949192
-(1 row)
-
+ERROR:  Altering a distributed sequence is currently not supported.
```

Comment:
I don't think we should remove the tests.  These are important tests for combinations prepared statements + sequences + local execution .

We should maybe just disable ddl propagation before alter

Reply:
I've removed the test as altering sequences are not supported for now. Let's discuss it further tomorrow to how we can do that test without altering sequence.

Reply:
set citus.enable_ddl_propagation TO OFF;
 ALTER SEQUENCE collections_list_key_seq NO MINVALUE NO MAXVALUE;
reset citus.enable_ddl_propagation;

with some comments should do the trick